Apps/Admin/Client/Pages/SupportPage.razor

```diff
@@ -86,7 +86,9 @@
                 <RowTemplate>
                     <MudTd DataLabel="HDID" data-testid=@($"user-table-hdid-{context.Hdid}")>@context.Hdid</MudTd>
                     <MudTd DataLabel="PHN" data-testid=@($"user-table-phn-{context.Hdid}")>@context.PersonalHealthNumber</MudTd>
-                    <MudTd DataLabel="Last Login" data-testid=@($"user-table-last-login-{context.Hdid}")>@DateFormatter.ToShortDateAndTime(context.LastLoginDateTime)</MudTd>
+                    <MudTd DataLabel="Last Login" data-testid=@($"user-table-last-login-{context.Hdid}")>
+                        @DateFormatter.ToShortDateAndTime(context.LastLoginDateTime.ToLocalTime())
```

Comment:
LastLoginDateTIme has already been converted to PST from UTC in changes made in AdminUserProfileMapUtils so is it necessary to use LastLoginDateTIme.ToLocalTime().  Also there is no guarantee the server will be running in PST.